### PR TITLE
GoLang basic & Networking phases

### PR DESCRIPTION
**Pull request by VitaliyShevchenko**
### Description
Added blocks for two first phases: GoLang basic and Networking
Added a description of news-aggregator project
Moved some details/materials from the previous go with clouds course

### CODE REVIEW

File: golang-basic/00-pre-requirements.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Pre requirements
+
+1. Tools needed for development: **go1.26**, **iTerm**, **git**, **IntelliJ IDEA**.
+2. IntelliJ IDEA should have next plugins installed:
```
Comment (Skillshot) [inline golang-basic/00-pre-requirements.md:4]: I will leave comments for myself to adjust this docs and you can merge in this way to get started with something.

So I guess now that we don't need Gradle and all it's better to tell to use GoLand or Idea with plugins (if they have a paid version). I guess community addition should suffice for them.

File: golang-basic/00-pre-requirements.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Pre requirements
```
Comment (Skillshot) [inline golang-basic/00-pre-requirements.md:1]: I guess we can make it an md book that they can access and serve from their local.

File: golang-basic/03-git-articles.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+### Description
+
+Develop API for news aggregator. This API will collect, process, and
```
Comment (dkalinin) [inline golang-basic/03-git-articles.md:3]: Should be about Git. This content should be moved to 02-apis.md, I guess.

File: golang-basic/04-clean-code.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+### Unit Tests
```
Comment (dkalinin) [inline golang-basic/04-clean-code.md:1]: Unit tests or clean code? Or both?

File: golang-basic/06-goroutines.md
Comment (dkalinin) [inline golang-basic/06-goroutines.md:1]: According to the schedule, Goroutines go after go templates.

**Review by dkalinin — COMMENTED** [review]
Also, we need to add task description for each step.

File: golang-basic/00-pre-requirements.md (hunk 2 above)
Comment (VitaliyShevchenko) [reply golang-basic/00-pre-requirements.md:1]: yes, or add a new tab ( `Docs`) to this project and host it by ourself

**Pull request merged**